tests/phpunit/Unit/functions/StrToCamelCaseTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Cig\Tests\Unit;
```

Comment:
I think we should caps the `src/functions` directory in this instance. If you do caps it, the commented out line in `src/index.php` with the path to that functions directory should be update too just in case. I always forget if it's Linux or macOS that is strict about path casing(I think macOS is the flexible one...).